tests/rptest/clients/rpk.py

```diff
@@ -287,7 +287,7 @@ def parse_field(field_name, string):
             return m['value']
 
         partition_pattern = re.compile(
-            "(?P<topic>.+) +(?P<partition>\d+) +(?P<offset>\d+) +(?P<log_end>\d+) +(?P<lag>\d+) *(?P<member_id>.*)? *(?P<client_id>.*)? *(?P<host>.*)?"
+            "(?P<topic>.+) +(?P<partition>\d+) +(?P<offset>\d+) +(?P<log_end>\d+) +(?P<lag>-?\d+) *(?P<member_id>.*)? *(?P<client_id>.*)? *(?P<host>.*)?"
```

Comment:
This change is in response to `rpk group describe` returning a negative number for lag when kafka is the source cluster.

See https://redpandadata.slack.com/archives/C01ND4SVB6Z/p1651692860593549?thread_ts=1651691083.566309&cid=C01ND4SVB6Z

This commit is just a placeholder until the above is resolved.

Reply:
GH issue here #4589